python/sglang/srt/models/llama4.py

```diff
@@ -55,7 +55,10 @@ def custom_routing_function(
         topk: int,
         renormalize: bool,
     ) -> Tuple[torch.Tensor, torch.Tensor]:
-        router_scores_aK, router_indices_aK = torch.topk(gating_output, topk, dim=-1)
+        assert topk == 1
+        router_scores_aK, router_indices_aK = torch.max(
```

Comment:
use if / else instead of assert

Reply:
OK, but then the "else" (topk) branch will never be executed in tests or real life